### PR TITLE
Add functionality to checkout a specific repo tag

### PR DESCRIPTION
**Pull request by Gabib32**
This runs a git checkout tags/<version> for the repo, so that
checks can be ran on more versions than latest. This is
currently being used to checkout different bundle definition
versions, but may be used by other repositories in the future.

Signed-off-by: gabrielle beyer <gabrielle.n.beyer@intel.com>

### CODE REVIEW

File: diva/fetch.go [hunk 1]
```diff
@@ -104,6 +104,32 @@ func FetchRepo(u UInfo) error {
 	return nil
 }
 
+// GetSpecificBundleTag clones the repo if it doesn't exist, and then checks
+// out the specific tag.
+func GetSpecificBundleTag(conf *config.Config, url, tag string) error {
```
Comment (matthewrsj) [inline diva/fetch.go]: You can drop the `Specific` wording here since it is still implied in `GetBundleTag`

Comment (matthewrsj) [reply diva/fetch.go]: Maybe `GetBundlesAtTag`?

File: diva/fetch.go [hunk 2]
```diff
@@ -104,6 +104,32 @@ func FetchRepo(u UInfo) error {
...
+		helpers.PrintBegin("cloning latest bundle definitions")
+		err := helpers.CloneRepo(url, filepath.Dir(conf.Paths.BundleDefsRepo))
+		if err != nil {
+			return err
+		}
+		helpers.PrintComplete("bundle repo cloned to %s", conf.Paths.BundleDefsRepo)
+		return nil
+	}
+
+	helpers.PrintBegin("checking out bundle definitions for tag/%s", tag)
```
Comment (matthewrsj) [inline diva/fetch.go]: This is very quick if the repo is already cloned. Please remove the prints.

**Comment by matthewrsj** [comment]
@Gabib32 does this PR need to be updated now that #65 has been merged?

**Comment by Gabib32** [comment]
Updated the PR to add the tag checkout to the pydeps command.

File: diva/fetch.go [hunk 3]
```diff
@@ -97,6 +97,30 @@ func FetchRepo(u UInfo) error {
...
+		helpers.PrintBegin("cloning latest bundle definitions")
+		err := helpers.CloneRepo(url, filepath.Dir(conf.Paths.BundleDefsRepo))
+		if err != nil {
+			return err
+		}
+		helpers.PrintComplete("bundle repo cloned to %s", conf.Paths.BundleDefsRepo)
+		return nil
+	}
+
+	err := helpers.CheckoutRepoTag(conf.Paths.BundleDefsRepo, tag)
```
Comment (matthewrsj) [inline diva/fetch.go]: just return here!

Comment (Gabib32) [reply diva/fetch.go]: YAAASSS Good call :+1000000:

**Pull request merged**